classes/file/FileManager.inc.php

```diff
@@ -549,7 +549,7 @@ function setMode($path, $mask) {
 	 */
 	function parseFileExtension($fileName) {
 		$fileParts = explode('.', $fileName);
-		if (is_array($fileParts)) {
+		if (is_array($fileParts) && count($fileParts) > 1) {
```

Comment:
Fixing a bug where if the file doesn't have an extension, basename was used instead of it